.goreleaser.yaml

```diff
@@ -0,0 +1,86 @@
+project_name: apm-lambda-extension
+
+before:
+  hooks:
+    - make check-extension
+    - make check-licenses
+    - make NOTICE.txt
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm64
+    flags:
+      - -trimpath
+    ldflags:
+      - -s -w
+    binary: "extensions/{{ .ProjectName }}"
+
+archives:
+  - id: zip
+    format: zip
+    name_template: |-
+      {{ .Tag }}-{{ .Os }}-{{ .Arch }}
+    files:
+      - NOTICE.txt
+      - dependencies.asciidoc
+    rlcp: true # https://goreleaser.com/deprecations/#archivesrlcp
```

Comment:
This option was not available in my current gorelease version:

```
   • releasing...     
   • loading config file       file=.goreleaser.yaml
   ⨯ release failed after 0.00s error=yaml: unmarshal errors:
  line 31: field rlcp not found in type config.Archive
make: *** [dist] Error 1
```

I've just updated the version with `go install github.com/goreleaser/goreleaser@latest`

I wonder if the version should be pinned in the makefile, see my proposal

Reply:
https://github.com/elastic/apm-aws-lambda/pull/362/commits/8bd2e4302b57a015a963ec39b16642bc946df466